metasrv/src/store/meta_raft_store.rs

```diff
@@ -80,7 +80,7 @@ pub struct MetaRaftStore {
     /// state machine is stored in another sled db since it contains user data and needs to be export/import as a whole.
     /// This db is also used to generate a locally unique id.
     /// Currently the id is used to create a unique snapshot id.
-    _db: sled::Db,
+    pub db: sled::Db,
```

Comment:
`pub(crate)` will let you be able to modify this field without introducing a breaking change:)